code/modules/antagonists/pirate/pirate_shuttle_equipment.dm

```diff
@@ -29,7 +29,7 @@
 		else
 			return
 	else
```

Comment:
do me a favor, convert this to early returns

Reply:
Yup